### PR TITLE
Forms implementation

### PR DESCRIPTION
**Pull request by Sick-Carlito**
# Dear reviewer

The following pull request is for implementing the following changes.

 - [x] HTML forms using Rails view helper methods and vanilla HTML.
 - [x] `new_user_path` points to the root (`/`) route.
 - [x] Users are now editable by going to the route `/users/:id/edit`.

If there is any problem let us know and we will kindly work on its solution. **Thanks for your valuable time**.

### CODE REVIEW

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle: 

Hi there team :+1:. Great job so far. But your project has some bugs that need to be fixed.

- [x] Please setup the code [linter](https://github.com/microverseinc/linters-config/tree/master/ruby). The linter is not only going to give a professional look to your repository but is also important to reduce errors and improve the overall quality of your code. You also need to fix any errors that may find.
Screenshot from 2019-12-11 09-14-29


- [x] It's a good practice to add a short summary of the changes made in the PR description. Take a time to read [this](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) Microverse article. It will give tips on how to make your project looks more professional. This will help you when looking for jobs. So just as it says in this article, we need to give a meaningful name for the PR and a short summary of the changes. You can still edit the PR description and gives a short summary.

- [x] You have extra 'end' in the update method from the User controller. This is causing an error
https://github.com/Sick-Carlito/re-former/blob/8cc92469b7955ea4ef92309241fb0258a874a88e/app/controllers/users_controller.rb#L20-L29

- [x] There is an additional parenthesis here in the pluralize method which is leading to an error. Delete the closing parenthesis after `@user.errors.count`
https://github.com/Sick-Carlito/re-former/blob/8cc92469b7955ea4ef92309241fb0258a874a88e/app/views/users/new.html.erb#L35-L38

- [x] You have the same pluralize error in user edit view
https://github.com/Sick-Carlito/re-former/blob/8cc92469b7955ea4ef92309241fb0258a874a88e/app/views/users/edit.html.erb#L2-L5

**[OPTIONAL]**
- [x]  Add a root path for the sign-up page to your routes file. This will make your page more similar to real-world webpages. 

Please kindly submit for code review again after making these changes
Happy coding :computer: 

**Sergio Torres**

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-    @user ||= User.new
+    @user = User.new
   end
 
   def create
     @user = User.new(user_params)
 
     if @user.save
-      redirect_to new_user_path
+      redirect_to root_path
```
Comment (adriaanbd) [inline app/controllers/users_controller.rb:12]: This should have redirected to `new_user_path` as you had it previously, please see the requirement [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) [`REQUIRED`]

**Review by adriaanbd — APPROVED** [review]
You project meets behavioural requirements and most functional requirements specified for this milestone. There is a small mistake made in a `redirect` as commented below, but it doesn't merit a failed review, since you've done a good job already addressing all changes, and it isn't breaking anything. Just make sure to read requirements carefully next time.

Status: `Approve`
Reviewer: `Adriaan Beiertz`

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (santiago-dev96) [reply app/controllers/users_controller.rb:12]: Thanks, I'll be more careful next time.

**Pull request merged**